packages/cms-cli/bin/cli.js

```diff
@@ -27,10 +28,16 @@ const listCommand = require('../commands/list');
 const openCommand = require('../commands/open');
 const mvCommand = require('../commands/mv');
 
+// Point update-notifier towards @hubspot/cli
+const pkg = { name: '@hubspot/cli', version };
```

Comment:
One thing I'm not clear on is if `cms-cli` version will continue to increment with `cli`? If so, we'll probably want to hard code this to a lower version number

Reply:
Plan is to continue using the same version scheme so that the tags / versions in the repo continue to be meaningful. We'll start with 3.0 for the `@hubspot/cli` release so that there is adequate space in the versioning scheme for `@hubspot/cms-cli` should a patch be needed.